roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,34 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
+- name: Creating Mux storage template
```

Comment:
is it possible to replace this `template` and `oc_obj` logic with a `oc_pvc` call?